ssl/ssl_lib.c

```diff
@@ -2277,39 +2277,41 @@ int SSL_get_key_update_type(const SSL *s)
     return s->key_update;
 }
 
-int SSL_renegotiate(SSL *s)
+/*
+ * Can we accept a renegotiation request?  If yes, set the flag and
+ * return 1 if yes. If not, raise error and return 0.
+ */
+static int can_renegotiation(SSL *s)
```

Comment:
`can_renegotiate` would sound more English to me. :grin: 

Also, I'd rather keep the duplication of `s->renegotiate = 1;` and keep the function read-only to align with the name.